transit.md

```diff
@@ -163,15 +165,19 @@ attempting to use the relay. If it has no viable direct hints, it will start
 using the relay right away. This prefers direct connections, but doesn't
 introduce completely unnecessary stalls.
 
-The Transit client can attempt connections to multiple relays, and uses the
-first one that passes negotiation. Each side combines a locally-configured
-hostname/port (usually "baked in" to the application, and hosted by the
-application author) with additional hostname/port pairs that come from the
-peer. This way either side can suggest the relays to use. The `wormhole`
-application accepts a `--transit-helper tcp:myrelay.example.org:12345`
-command-line option to supply an additional relay. The connection hints
-provided by the Transit instance include the locally-configured relay, along
-with the dynamically-determined direct hints. Both should be delivered to the
+The Transit client attempts connections to multiple relays, and uses
+the first one that passes negotiation. Each side combines a
+locally-configured hostname/port (usually "baked in" to the
+application, and possibly hosted by the application author) with
+additional hostname/port pairs that come from the peer. This way
+either side can suggest the relays to use. The `wormhole` application
+accepts configuration switch that takes as input, the relay server's
+address, for eg: `tcp:myrelay.example.org:12345` or
```

Comment:
My understanding is that `e.g.` means ["for example"](https://en.wikipedia.org/wiki/List_of_Latin_phrases_(E)#e.g.), making the "for" redundant.
```suggestion
address, e.g.: `tcp:myrelay.example.org:12345` or
```